tracing/opencensus/endpoint_options.go

```diff
@@ -11,6 +15,16 @@ type EndpointOptions struct {
 	// Attributes holds the default attributes which will be set on span
 	// creation by our Endpoint middleware.
 	Attributes []trace.Attribute
+
+	// GetName holds the function used for generating the span name
+	// based on the current name and from the information found in the incoming Request.
+	//
+	// If the returned name is empty, the existing name for the endpoint is kept.
+	GetName func(ctx context.Context, name string) string
```

Comment:
Is "the existing name for the endpoint" what is passed for the name parameter to GetName? If so,

> GetName is an optional function that can set the span name based on the existing name for the endpoint and information in the context. If the function is nil, or the returned name is empty, the existing name for the endpoint is used.